snc.sh

```diff
@@ -54,17 +61,9 @@ function run_preflight_checks() {
                 preflight_failure  "libvirtd is not listening for plain-text TCP connections, see https://github.com/openshift/installer/tree/master/docs/dev/libvirt#configure-libvirt-to-accept-tcp-connections"
         fi
 
-        # check if libvirtd has access to an x86_64 hypervisor
-        local arch
-        arch=$(virsh -c ${LIBVIRT_URI} capabilities | ${XMLLINT} --xpath '/capabilities/host/cpu/arch/text()' -)
```

Comment:
maybe keep this check anyway for x86,ppc64le and s390x so other arches like arm will be excluded (till we have support)

Reply:
Yeah, would be good to keep such a warning (or even failure, as it's only a warning to avoid blocking ppc64 efforts)

Reply:
Please add a switch like structure to determine the ARCH and if supported.